### PR TITLE
various: smash overly-long lines

### PR DESCRIPTION
**Pull request by DomT4**
Call your representatives today and urge them to abandon the _"War on Drugs"_ & instead join the _"War on >150 character lines!"_.

img.

### CODE REVIEW

**Comment by DomT4** [comment]
Nothing unexpected amongst the failures, for better or worse. Merged.

94 non-exempt lines over 150 characters remain... for now 🗡.

File: Formula/redpen.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ def install
   test do
     path = "#{libexec}/sample-doc/en/sampledoc-en.txt"
     output = "#{bin}/redpen -l 20 -c #{libexec}/conf/redpen-conf-en.xml #{path}"
-    assert_match /^sampledoc-en.txt:1: ValidationError[SymbolWithSpace]*/, shell_output(output).split("\n").select { |line| line.start_with?("sampledoc-en.txt") }[0]
+    match = /^sampledoc-en.txt:1: ValidationError[SymbolWithSpace]*/
+    assert_match match, shell_output(output).split("\n").select { |line| line.start_with?("sampledoc-en.txt") }[0]
```
Comment (claui) [inline Formula/redpen.rb]: `.find { … }` instead of `select { … }.[0]` while we’re at it?

File: Formula/woboq_codebrowser.rb [hunk 2]
```diff
@@ -13,13 +13,18 @@ class WoboqCodebrowser < Formula
...
   depends_on "cmake" => :build
+  depends_on "llvm"
 
   def install
-    system "cmake", ".", "-DLLVM_CONFIG_EXECUTABLE=#{Formula["llvm"].opt_bin}/llvm-config", "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON", *std_cmake_args
+    args = std_cmake_args + %W[
+      -DLLVM_CONFIG_EXECUTABLE=#{Formula["llvm"].opt_bin}/llvm-config
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+    ]
+    system "cmake", ".", *args
```
Comment (claui) [inline Formula/woboq_codebrowser.rb]: Will `cmake` still do the same thing when `std_cmake_args` comes before the `-D`… constants?

File: Formula/mdxmini.rb [hunk 3]
```diff
@@ -59,11 +61,19 @@ def install
...
-    expected = "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         \x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{"
+    # Trailing whitespace is intentional & shouldn't be removed.
+    l1 = "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         "
+    l2 = "\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          "
+    expected = <<~EOS
+      #{l1}
+      #{l2}
+      (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{
+    EOS
     expected.force_encoding("ascii-8bit") if result.respond_to? :force_encoding
```
Comment (claui) [inline Formula/mdxmini.rb]: I feel the following may be a bit easier to read:

```
expected = [
             "\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B",
             "\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC",
             " (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{",
           ].join("         ")
```

…

```
assert_equal expected, result
```

___

Or alternatively, inline the `l1` and `l2` constants:

```
expected = <<~EOS
  #{"\x82\xDB\x82\xC1\x82\xD5\x82\xE9\x83\x81\x83C\x83\x8B         "}
  #{"\x83o\x83b\x83N\x83A\x83b\x83v\x8D\xEC\x90\xAC          "}
  (C)Falcom 1992 cv.\x82o\x82h. ass.\x82s\x82`\x82o\x81{
EOS
```

**Comment by claui** [comment]
Oops. Ninja merge? Guess I should speed up my reviewing game 😄

**Comment by claui** [comment]
> 94 non-exempt lines over 150 characters remain... for now 🗡.

Keep fighting the good fight!

File: Formula/mdxmini.rb (hunk 3 above)
Comment (DomT4) [reply Formula/mdxmini.rb]: If you want to follow up I'm open to the latter, as long as audit doesn't take a shot at it!

File: Formula/woboq_codebrowser.rb (hunk 2 above)
Comment (DomT4) [reply Formula/woboq_codebrowser.rb]: Aye, we use this syntax plenty in core without issue 👍.

**Comment by DomT4** [comment]
> Oops. Ninja merge? 

Heh, apologies. Core has taught me the longer a PR that touches multiple files sits the more time-consuming rebasing you're imminently going to have to do so I tend to merge ASAP to avoid that 😅.

If there's stuff you want to follow-up on you're more than welcome to. Just trying to make progress here whilst not letting it eat up too much time, given the wad of other stuff I want to get through 🙈.

**Comment by claui** [comment]
No hard feelings, and good point about the multi-file-PR thing. My nits were super minor anyway, so 👍

File: Formula/mdxmini.rb (hunk 3 above)
Comment (claui) [reply Formula/mdxmini.rb]: > as long as audit doesn't take a shot at it!

It does 😒

May as well stay as is, then! 😄

Comment (DomT4) [reply Formula/mdxmini.rb]: Audit likes to get _real punchy_ with whitespace 😸

**Comment by DomT4** [comment]
Heh, it makes a pleasant change for me to not be the one nudging people to perfect style a little bit more! Thanks for the review 🙇

File: Formula/mdxmini.rb (hunk 3 above)
Comment (claui) [reply Formula/mdxmini.rb]: I didn’t even get that far! They pulled me over due to “literal interpolation” already.

**Pull request closed without merging**